lib/resources/company.js

```diff
@@ -172,10 +172,28 @@ const emailTemplates = {
     request.put(opts, routes.company.emailTemplates(body.id), body),
 }
 
+
+/**
+ * `POST /company/email_confirmation`
+ * Send email confirmation a company from the given payload.
```

Comment:
Aqui acho que ficaria melhor descrito assim:

```suggestion
 * Resend account confirmation email to a previously registered company.
```